### PR TITLE
fix: arm containerd no match container id

### PR DESCRIPTION
**Pull request by tomatopunk**
#### What this PR does / why we need it:

arm containerd no match container id

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    arm containerd no match container id          |
| 🇨🇳 中文    |     修复 arm containerd 环境无法匹配到的 container id 问题         |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: cmd/monitor/collector/bootstrap.yaml [hunk 1]
```diff
@@ -191,7 +191,8 @@ erda.oap.collector.processor.modifier@docker_container_summary:
       value: fields.tx_errors
     - action: regex
       key: tags.id
-      value: '^\/kubepods.*\/.*?\/.*?\/(\w+\-)?(?P<container_id>\w+)'
+      value: '\/kubepods.*?(?P<container_id>\w{64})$' #  have kubepods and match 64-character
+      # containerd arm: /system.slice/containerd.service/kubepods-burstable-pod29c95a11_3435_474a_be34_226976d3035f.slice:cri-containerd:01c5cfd0633692a2a14bfa656e947cd4c23e5d17492382f354f262c72f27802d
       # /kubepods.slice/kubepods-burstable.slice/kubepods-burstable-podfebb6bd8_349e_49ea_a629_bf27a28d63f3.slice/docker-39693d24c3985d77348b260bdb4f5af465fc0d86b3c07ec8560b3b56ae629d62.scope
```
Comment (sfwn) [inline cmd/monitor/collector/bootstrap.yaml:196]: this case doesn't match
image

**Review by sfwn — CHANGES_REQUESTED** [review]

File: cmd/monitor/collector/bootstrap.yaml (hunk 1 above)
Comment (tomatopunk) [reply cmd/monitor/collector/bootstrap.yaml:196]: In fact, the original expr can't parse this structure, maybe it doesn't need to be dealt with

Comment (sfwn) [reply cmd/monitor/collector/bootstrap.yaml:196]: just remove this unused case

**Review by sfwn — APPROVED** [review]

**Pull request merged**